application/src/main/java/uk/nhs/hee/tis/revalidation/repository/RecommendationElasticSearchRepository.java

```diff
@@ -33,23 +33,25 @@
 public interface RecommendationElasticSearchRepository extends
     ElasticsearchRepository<RecommendationView, String> {
 
-  @Query("{\"bool\":{\"filter\":[{\"match\":{\"underNotice\":\"YES\"}},{\"match\":"
-      + "{\"designatedBody\":\"?1\"}},{\"match\":{\"existsInGmc\":\"true\"}},{\"bool\":"
-      + "{\"should\":[{\"wildcard\":{\"doctorFirstName\":{\"value\":\"?0*\"}}},{\"wildcard\":"
+  @Query("{\"bool\":{\"filter\":[{\"match\":{\"underNotice\":\"YES\"}},{\"match\":{"
+      + "\"designatedBody\":\"?1\"}},{\"match\":{\"existsInGmc\":\"true\"}},{\"bool\":{"
+      + "\"should\":[{\"wildcard\":{\"doctorFirstName\":{\"value\":\"?0*\"}}},{\"wildcard\":"
       + "{\"doctorLastName\":{\"value\":\"?0*\"}}},{\"wildcard\":{\"gmcReferenceNumber\":"
-      + "{\"value\":\"?0*\"}}}]}},{\"match_phrase\":{\"programmeName\":{\"query\":\"?2\","
-      + "\"zero_terms_query\":\"all\"}}}]}}")
+      + "{\"value\":\"?0*\"}}}]}},{\"match_phrase\":{\"programmeName\":"
+      + "{\"query\":\"?2\",\"zero_terms_query\":\"all\"}}},{\"match_phrase\":"
+      + "{\"gmcStatus\":{\"query\":\"?3\",\"zero_terms_query\":\"all\"}}}]}}")
```

Comment:
Looks like a large diff for adding 1 field.  Will check back when I've got a bit more mental capacity if it hasn't been merged.

Reply:
I think I just put the carriage returns in a bit differently, will try and get to match the previous a bit more closely to make the diff more obvious